pkgs/applications/editors/emacs/default.nix

```diff
@@ -2,6 +2,7 @@
 , pkgconfig, gettext, libXft, dbus, libpng, libjpeg, libungif
 , libtiff, librsvg, gconf, libxml2, imagemagick, gnutls, libselinux
 , alsaLib, cairo, acl, gpm, cf-private, AppKit, GSS, ImageIO, m17n_lib, libotf
+, xorg
```

Comment:
Please don't add the entire `xorg` package set as a function parameter. Please take only those packages as an argument that you actually use, i.e. `libXcursor`.